docs/schema/reference.md

```diff
@@ -226,6 +226,8 @@ Information on subcontracts is not currently included in the core OCDS schema, b
 
 The transaction block is modelled on the [International Aid Transparency Initiative (IATI) transaction element](https://iatistandard.org/en/iati-standard/203/activity-standard/iati-activities/iati-activity/transaction/), and can be used to represent actual flows of money between organizations in relation to this contract. As with the [budget](#budget) block, this can be used to cross-reference to a third party `source` of data, and ought to re-use identifiers from that source.
 
+To represent planned payments, use [Milestones](#milestones).
```

Comment:
Move this after the "In most circumstances" paragraph, as this paragraph leads the reader away from Transactions, whereas "In most circumstances" is still about transactions. We might prefer to put this in a `{note}` since it's not directly about transactions.

Also, let's use this PR to move the jsonschema directive below the paragraphs, just above the extensionlist.
```suggestion
To represent planned payments, use [Milestones](#milestones) instead.
```